cmd/root.go

```diff
@@ -60,6 +60,9 @@ func AddCommands(cmd *cobra.Command) {
 	cmd.PersistentFlags().IntVar(&config.SyncRequestTimeout, "sync-request-timeout",
 		2000, "the timeout in ms for direct requests in the sync")
 
+	cmd.PersistentFlags().IntVar(&config.AtxsSelectCount, "atxs-select-count",
```

Comment:
Let's call it `AtxsPerBlock` or something?

Reply:
Done